doc/source/user_guide/numpy_images.rst

```diff
@@ -2,78 +2,77 @@
 A crash course on NumPy for images
 ==================================
 
-Images manipulated by ``scikit-image`` are simply NumPy arrays. Hence, a
-large fraction of operations on images will just consist in using NumPy::
+Images in ``scikit-image`` are represented by NumPy ndarrays. Hence, many 
+common operations can be achieved using standard NumPy methods for 
+manipulating arrays:
```

Comment:
I'm pretty sure these need to be double `::` to indicate a code block? @emmanuelle? See the sphinx ReST [guide](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html)

Reply:
Oh LOL your identical comment appeared in my inbox just now!